src/test/java/com/google/step/youtube/DescriptionParserTest.java

```diff
@@ -1,12 +1,201 @@
 package com.google.step.youtube;
 
-import org.junit.Assert;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
```

Comment:
Put static imports and regular imports in separate blocks.  With static going first.
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Done.